runners/java-fn-execution/src/test/java/org/apache/beam/runners/fnexecution/wire/CommonCoderTest.java

```diff
@@ -382,6 +384,17 @@ private static Object convertValue(Object value, CommonCoder coderSpec, Coder co
       Map<String, Object> kvMap = (Map<String, Object>) value;
       Coder windowCoder = ((TimestampPrefixingWindowCoder) coder).getWindowCoder();
       return convertValue(kvMap.get("window"), coderSpec.getComponents().get(0), windowCoder);
+    } else if (s.equals(getUrn(StandardCoders.Enum.NULLABLE))) {
```

Comment:
Are we generating individual tests for coders here or are we lumping everything into a single test ? If it's latter probably we should consider adding tests specifically for the new case.

Reply:
It is a single test in the java sense, but there are 3 test cases in the yaml file for nullable coders.